File: library/src/main/java/zjm/cst/dhu/library/mvp/presenter/BasePresenter.java
package zjm.cst.dhu.library.mvp.presenter;

import rx.subscriptions.CompositeSubscription;
import zjm.cst.dhu.library.mvp.view.base.BaseView;

/**
 * Created by zjm on 2017/5/9.
 */

public interface BasePresenter<T extends BaseView> {

    void attachView(T BaseView);

    void detachView();
}

File: library/src/main/kotlin/zjm/cst/dhu/library/mvp/presenter/BasePresenter.kt
package zjm.cst.dhu.library.mvp.presenter

import zjm.cst.dhu.library.mvp.view.base.BaseView

/**
 * Created by zjm on 2017/5/9.
 */

interface BasePresenter<in T : BaseView> {

    fun attachView(BaseView: T)

    fun detachView()
}
